CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+## Unreleased
+### Added
+- Basic ExDoc configuration
+- Markdown documentation (README, LICENSE, CHANGELOG)
```

Comment:
I think you can leave this out until you release a version. Also, I'd suggest to only include things here that are pertintent to know for users who are upgrading versions. For example, if I'm upgrade from 0.1.0 to 1.0.0, I probably don't care about whether or not the README changed, and I"m mouch more interested in how the interfaced changed and what will break.